src/pages/SetupPlot.vue

```diff
@@ -173,7 +172,7 @@ export default defineComponent({
     },
     stats(): StatsType {
       const totalDiskSizeGB = util.toFixed(this.driveStats.totalBytes / 1e9, 2)
-      const safeAvailableGB = this.driveStats.freeBytes / 1e9
+      const safeAvailableGB = this.driveStats.freeBytes / 1e9 - 20
```

Comment:
What is this 20? We need to allocate 10GB for node and the rest for farmer, but the user should only specify the combined number, they only care how much will software use in total.

Reply:
We have talked with you previously that node is at most occupy 20GB. 
So I put 20GB for a safe cap. 

Jeremiah and I discussed that we should not complicate the user workflow and maybe should not even mention about that (node occupying space). So user can enter the plot size, and we can deduct 20GB from the safe space.

I can update this to 10 if you like, no problem. 
However, when we switch to full-node, will it still occupy around 10GB? I think it would be around 1-2GB at most? Am I recalling it wrong?